packages/deployer/terraform/airnode/modules/function/main.tf

```diff
@@ -12,8 +12,8 @@ EOC
   }
 
   triggers = {
-    source_file_hash        = fileexists("${local.tmp_source_dir}/${var.source_file}") ? filesha256(var.source_file) : ""
-    configuration_file_hash = fileexists("${local.tmp_configuration_dir}/${var.configuration_file}") ? filesha256(var.configuration_file) : ""
+    // Run always
+    trigger = uuid()
```

Comment:
What is this for?

Reply:
I needed a way for that `null_resource` to run every time recipes are run. Well, at first I wanted to do it in a clever way, so it will run only if something changes, but that was not working for the Docker version in a scenario where the state already exists but the temp directories don't. So I decided to run it every time. It does so little that it should not be a problem.